src/MobileUI/ViewModels/ProfileViewModels/MyProfileViewModel.cs

```diff
@@ -1,106 +1,51 @@
-﻿using CommunityToolkit.Mvvm.Messaging;
+﻿using System.Reactive.Linq;
+using CommunityToolkit.Mvvm.Messaging;
 using SSW.Rewards.Mobile.Messages;
-using System.Collections.ObjectModel;
 
 namespace SSW.Rewards.Mobile.ViewModels.ProfileViewModels;
 
 public class MyProfileViewModel(
     IRewardService rewardsService,
     IUserService userService,
-    ISnackbarService snackbarService,
     ILeaderService leaderService,
     IDevService devService,
     IPermissionsService permissionsService)
-    : ProfileViewModelBase(rewardsService, userService, snackbarService, devService, permissionsService),
-        IRecipient<ProfilePicUpdatedMessage>,
-        IRecipient<PointsAwardedMessage>,
-        IRecipient<SocialUsernameAddedMessage>
+    : ProfileViewModelBase(rewardsService, userService, devService, permissionsService),
+        IRecipient<PointsAwardedMessage>
```

Comment:
Should this still be implementing this `IRecipient`? I think the points change is included in this PR isn't it?

Reply:
Yes, now we can remove this message. Thanks.